src/core/celestials/pelle/galaxy-generator.js

```diff
@@ -102,7 +102,7 @@ export const GalaxyGenerator = {
     );
 
     if (!this.capRift) {
-      PelleRifts.all.forEach(r => r.reducedTo = Math.min(r.reducedTo + 0.03 * diff / 1000, 2));
+      PelleRifts.all.forEach(r => r.reducedTo = Decimal.add(r.reducedTo, diff.div(1e5).mul(3)).min(2).toNumber());
```

Comment:
Codacy emits an error here: `The numeric literal '1e5' will have at different value at runtime.`, but that may be a false positive given my testing indicates this line works as expected.
See also https://github.com/pmd/pmd/issues/4345.

Reply:
Codacy generally has quite a few false-positives when it comes to numeric literals, it can generally be ignored